crates/cosmos/cosmos-wasm-relayer/tests/cosmos_to_wasm_cosmos.rs

```diff
@@ -66,6 +66,19 @@ fn test_cosmos_to_wasm_cosmos() -> Result<(), Error> {
             hasher.finalize().into()
         };
 
+        let wasm_additional_byte_code = match var("ADDITIONAL_WASM_FILE_PATH") {
+            Ok(paths_str) => {
+                let paths: Vec<PathBuf> = paths_str.split(',').map(PathBuf::from).collect();
+
+                let mut byte_code = Vec::with_capacity(paths.len());
+                for path in paths {
+                    byte_code.push(tokio::fs::read(path).await?);
+                }
+                byte_code
```

Comment:
same here. you may want to use `std::fs::read`.

Reply:
b36edc08b76668477dced47d9b96ca9fd9fb2617